torch/nn/functional.py

```diff
@@ -1148,6 +1148,19 @@ def rrelu(input, lower=1. / 8, upper=1. / 3, training=False, inplace=False):
 See :class:`~torch.nn.LogSigmoid` for more details.
 """)
 
+@weak_script
+def gelu(input):
+    r"""gelu(input) -> Tensor
+
+    Applies element-wise the function
+    :math:`\text{GeLU}(x) = x * \Phi(x)`
+
+    where `\Phi(x)` Cumulative Distribution Function for Gaussian Distribution.
+
+    See :`Gaussian Error Linear Units (GELUs) <https://arxiv.org/abs/1606.08415>`.
```

Comment:
add an entry in https://raw.githubusercontent.com/pytorch/pytorch/master/docs/source/nn.rst similar to elu:

```
:hidden:`gelu`
~~~~~~~~~~~~~

.. autofunction:: gelu
```

Reply:
Thanks! Done.